### PR TITLE
Add libredwg

### PR DESCRIPTION
**Pull request by brennanleblanc0**

### CODE REVIEW

File: libredwg.mk [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+libredwg-package: libredwg-stage
+    # libredwg.mk Package Structure
+	rm -rf $(BUILD_DIST)/libredwg{0,-dev,-utils}
+	mkdir -p $(BUILD_DIST)/libredwg0/usr/lib \
+			$(BUILD_DIST)/libredwg-dev/usr/{lib,bin} \
+			$(BUILD_DIST)/libredwg-utils/usr
+    
+    # libredwg.mk Prep libredwg
+	cp -a $(BUILD_STAGE)/libredwg/usr/lib/libredwg.0.dylib $(BUILD_DIST)/libredwg/usr/lib
```
Comment (Diatrus) [inline libredwg.mk:37]: `$(BUILD_DIST)/libredwg/usr/lib` isn't a directory.

File: libredwg.mk [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	cp -a $(BUILD_STAGE)/libredwg/usr/{bin,share} $(BUILD_DIST)/libredwg-utils/usr
+	cp -a $(BUILD_STAGE)/libredwg/usr/include $(BUILD_DIST)/libredwg-dev/usr
+	cp -a $(BUILD_STAGE)/libredwg/usr/lib/{pkgconfig,libredwg.{a,dylib}} $(BUILD_DIST)/libredwg-dev/usr/lib
+    
+    # libredwg.mk Sign
+	$(call SIGN,libredwg-utils,general.xml)
```
Comment (Diatrus) [inline libredwg.mk:43]: The dynamic library package needs to be signed as well.

File: build_info/libredwg-dev.control [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Package: libredwg-dev
+Version: @DEB_LIBREDWG_V@
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
+Section: Development
+Homepage: https://www.gnu.org/software/libredwg/
+Description: GNU LibreDWG is a free C library to handle DWG files. (Development files)
+Depends: libredwg0 (= @DEV_LIBREDWG_V@)
```
Comment (Diatrus) [inline build_info/libredwg-dev.control:8]: You put DEV instead of DEB here?

File: build_info/libredwg-utils.control [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Package: libredwg-utils
+Version: @DEB_LIBREDWG_V@
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
+Section: Utilities
+Homepage: https://www.gnu.org/software/libredwg/
+Description: GNU LibreDWG is a free C library to handle DWG files. (Utility Files)
+Depends: libredwg0 (= @DEV_LIBREDWG_V@)
```
Comment (Diatrus) [inline build_info/libredwg-utils.control:8]: You put DEV instead of DEB here?

**Comment by brennanleblanc0** [comment]
Packages fine now, sorry about that!

**Pull request merged**